src/app/table/view/header/header-cell.component.html

```diff
@@ -34,7 +34,7 @@
       <div *ngFor="let child of columnInfo.subcolumnsChildren" style="display: table-row">
         <div (click)="onSortClick(child)" [class.clickable]="child.sortable" id="sort-child">
           <ng-template *ngIf="child.contentTemplateRef" [ngTemplateOutlet]="child.contentTemplateRef"></ng-template>
-          <span *ngIf="!child.contentTemplateRef">{{ child.caption }}</span>
+          <span style="word-break: nowrap" *ngIf="!child.contentTemplateRef">{{ child.caption }}</span>
```

Comment:
Minor nitpick - put the `ngIf` before the `style` attribute - it's more important not to miss it when reading the template